ttnn/cpp/ttnn/operations/conv/conv2d/conv2d_utils.cpp

```diff
@@ -547,7 +547,9 @@ std::tuple<ttnn::Shape, ttnn::MemoryConfig> determine_input_memory_config(
     return {input_padded_shape, input_tensor_sharded_memory_config};
 };
 
-static std::tuple<ttnn::Shape, ttnn::MemoryConfig, bool> get_conv_padded_input_shape_and_mem_config(
+namespace {
+
+std::tuple<ttnn::Shape, ttnn::MemoryConfig, bool> get_conv_padded_input_shape_and_mem_config(
```

Comment:
can't we just keep this method be no member static function, why do we need anonymous namespace?
It already in ttnn::operations:con namespace